File: Array/04.FizzBuzz/FizzBuzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        return ['Fizz' * (not i % 3) + 'Buzz' * (not i % 5) or str(i) for i in xrange(1, n + 1)]

File: Array/FizzBuzz/FizzBuzz.cpp

class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> res(n, "");
        for(int i = 1; i<=n; ++i) {
            bool multiplesOfThree = (i % 3 == 0);
            bool multiplesOfFive = (i % 5 == 0);

            if(multiplesOfThree && multiplesOfFive) { res[i-1] = "FizzBuzz"; }
            else if(multiplesOfThree) { res[i-1] = "Fizz"; }
            else if(multiplesOfFive) {res[i-1] = "Buzz"; }
            else { res[i-1] = to_string(i); }
        }
        return res;
    }
};